[2024-12-29T18:36:20.583+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-29T18:36:20.764+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bash_dag.bash_ex2 scheduled__2024-12-26T08:00:00+00:00 [queued]>
[2024-12-29T18:36:20.799+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bash_dag.bash_ex2 scheduled__2024-12-26T08:00:00+00:00 [queued]>
[2024-12-29T18:36:20.800+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-29T18:36:20.880+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): bash_ex2> on 2024-12-26 08:00:00+00:00
[2024-12-29T18:36:20.906+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=101) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-29T18:36:20.916+0000] {standard_task_runner.py:63} INFO - Started process 114 to run task
[2024-12-29T18:36:20.917+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bash_dag', 'bash_ex2', 'scheduled__2024-12-26T08:00:00+00:00', '--job-id', '1152', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmp1mile265']
[2024-12-29T18:36:20.923+0000] {standard_task_runner.py:91} INFO - Job 1152: Subtask bash_ex2
[2024-12-29T18:36:21.055+0000] {task_command.py:426} INFO - Running <TaskInstance: bash_dag.bash_ex2 scheduled__2024-12-26T08:00:00+00:00 [running]> on host 87a1e11545b0
[2024-12-29T18:36:21.409+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bash_dag' AIRFLOW_CTX_TASK_ID='bash_ex2' AIRFLOW_CTX_EXECUTION_DATE='2024-12-26T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-26T08:00:00+00:00'
[2024-12-29T18:36:21.409+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-29T18:36:21.463+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-29T18:36:21.466+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Task 2 now $(date)"']
[2024-12-29T18:36:21.516+0000] {subprocess.py:86} INFO - Output:
[2024-12-29T18:36:21.538+0000] {subprocess.py:93} INFO - Task 2 now Sun Dec 29 18:36:21 UTC 2024
[2024-12-29T18:36:21.543+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-29T18:36:21.544+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-29T18:36:21.618+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bash_dag, task_id=bash_ex2, run_id=scheduled__2024-12-26T08:00:00+00:00, execution_date=20241226T080000, start_date=20241229T183620, end_date=20241229T183621
[2024-12-29T18:36:21.709+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-29T18:36:21.750+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-29T18:36:21.755+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
